src/components/Kv/KvChipClassic.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+	<button class="tw-rounded tw-bg-tertiary tw-p-1 tw-uppercase tw-text-small tw-shrink-0">
+		<slot></slot> X
+	</button>
+</template>
+
+<script>
+export default {
+	name: 'KvChipClassic',
+	props: {
+		// eslint-disable-next-line vue/require-default-prop
+		items: {
+			type: Array,
+			required: false,
+		},
+	},
+	methods: {
+		onChange(event) {
```

Comment:
I would convert this to `onClick` for the method name, `click` in the emit below and then you'll want to add `@click="onClick"` to your `button` element above.